plugin/evm/upgrade/legacy/params.go

```diff
@@ -0,0 +1,7 @@
+package legacy
```

Comment:
does this need a header?

Reply:
Yes it does, but that's coming from the master branch. I'll create a PR to add it to master 😉